### DISCUSSION TITLE
Option To Disable Villager Trading On Islands

### DISCUSSION THREAD

**Discussion by rrodg12** [BG-Software-LLC/SuperiorSkyblock2#560] [Denied Feature Requests] [answered]
Add A Way In The Configuration To Disable Villager Trading On Islands So Players Cant Easlily Get Emeralds For Island Level.

**Comment by OmerBenGera** [comment, marked as answer]
This doesn't sound like a feature that SSB should implement.
Look for other plugins that add this restrict villagers.
